src/Microsoft.AspNet.Server.Kestrel/Http/SocketOutput.cs

```diff
@@ -312,10 +302,6 @@ private void OnWriteCompleted(WriteContext writeContext)
                 {
                     scheduleWrite = true;
```

Comment:
Isn't this now redundant? Are we already guaranteed to have a queued write operation that will handle the `_nextWriteContext` based on the logic in WriteAsync?